### ISSUE TITLE
Support "form" workflow

### ISSUE THREAD

**Issue by MarcSkovMadsen** [jmosbacher/pydantic-panel#11]
Sometimes you don't want your model to change as soon as the value in your widget changes.

You might want to change several values before you press a *submit* button. Optionally some validation would run before changing the values of the underlying model.

It would be nice if this workflow was supported. It might be as simple as providing an example in the documentation on how to do this.

**Comment by jmosbacher** [comment]
Yeah this is definitely doable. I could add a `mode` attribute to the PydanticModelEditor widget that would then add a _submit_ button at the end when if form mode.
